cli/command/manifest/cmd.go

```diff
@@ -0,0 +1,45 @@
+package manifest
+
+import (
+	"fmt"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+
+	"github.com/spf13/cobra"
+)
+
+// NewManifestCommand returns a cobra command for `manifest` subcommands
+// nolint: interfacer
+func NewManifestCommand(dockerCli *command.DockerCli) *cobra.Command {
+	// use dockerCli as command.Cli
+	cmd := &cobra.Command{
+		Use:   "manifest COMMAND",
+		Short: "Manage Docker image manifests and lists",
+		Long:  manifestDescription,
+		Args:  cli.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Fprintf(dockerCli.Err(), "\n"+cmd.UsageString())
+		},
+	}
+	cmd.AddCommand(
+		newCreateListCommand(dockerCli),
+		newInspectCommand(dockerCli),
+		newAnnotateCommand(dockerCli),
+		newPushListCommand(dockerCli),
```

Comment:
It seems like maybe we are missing the simple `docker manifest list` command? I looke for it to check that it supported filters and formats, and can't find it at all.

Reply:
are you looking for `docker manifest list ... ` ?
p.s. sorry for the delay on responding to your comments! i've been saving up the doc stuff